### ISSUE TITLE
Database User/Pass Rotation

### ISSUE THREAD

**Issue by hongkongkiwi** [FusionAuth/fusionauth-issues#973]
Hi there,

I'm interested in using AWS Secrets Manager (along with Kubernetes) to automatically rotate the database user/password every 30 days.

I'm keen to keep the docker running, so ideally I would just update the configuration file and tell FusionAuth to reload it. Any database connections which need renewing should use the new user/password and any existing connections should stay connected.

Is there a way to achieve this? Or if not whats the best way to handle automatic db password rotation (such as with AWS Secrets Manager with a lambda function).

**Comment by mooreds** [comment]
@hongkongkiwi This is a great question for our forums. Or, if you have a paid edition, please open a support ticket.

We aim to keep github focused on bug reports and feature requests.

Please head over to https://fusionauth.io/community/forum/ and post your question there.

**Comment by hongkongkiwi** [comment]
Ok, let me rephrase this as a feature request. Support Reload DB credentials via Sighup for credential rotation.

Or if this is currently possible please add to documentation.

**Comment by mooreds** [comment]
Sure, I'll leave this as a feature request.

**Comment by robotdan** [comment]
Can you just restart the fusionauth service? If you are in a multi-node configuration, this should be roughly equivalent.
